services/gitlab/gitlab-tag.service.js

```diff
@@ -0,0 +1,62 @@
+import Joi from 'joi'
+import { optionalUrl } from '../validators.js'
+import { NotFound } from '../index.js'
+import GitLabBase from './gitlab-base.js'
+
+const schema = Joi.array().items(
+  Joi.object({
+    name: Joi.string().required(),
+  })
+)
+
+const queryParamSchema = Joi.object({
+  gitlab_url: optionalUrl,
+}).required()
```

Comment:
The `gitlab_url` param isn't showing up in the builder/builder modal in the front-end. I think we need a second example for the case where `gitlab_url` is set to enable that.

Reply:
Fixed in 8d5cfe16691522fa703bafa16e90d0a5c3cb797f